### DISCUSSION TITLE
The remote server is not a valid.[Fixed]

### DISCUSSION THREAD

**Discussion by macbethsoftwares** [saran-surya/email_auth#23] [Q&A] [answered]
Hello. I have followed all the instructions said setup server(https://saran-surya.github.io/email-auth-node/) . but still getting these errors.

--- Package Error ---
Unable to access remote server. 😑
--- End Package Error ---

--- package Error ---

email-auth >> The remote server is not a valid.
email-auth >> configured server : "server link"
--- package Error ---

**Comment by saran-surya** [comment, marked as answer]
Hi, so seems like you have not provided the server url of the production server., As you can see the configured server is returning 'server url', Kindly pass in the home page of the server url, until .com or .co and that should be fine.
